mirgecom/initializers.py

```diff
@@ -54,60 +54,62 @@
 from pytools.obj_array import make_obj_array
 from mirgecom.eos import IdealSingleGas
 from mirgecom.fluid import make_conserved
+from mirgecom.gas_model import make_fluid_state
 
 
-def initialize_flow_solution(actx, dcoll, gas_model, dd_bdry=None, pressure=None,
-                             temperature=None, density=None, velocity=None,
-                             mass_fractions=None, make_fluid_state=False):
+def initialize_flow_solution(actx, nodes, gas_model=None, eos=None,
```

Comment:
I don't think I like `nodes` as an argument name.  I know the original used "nodes" but that was bad, too.   Can we just call it `coords` or some such?

Reply:
Renamed to `coords`.